cuda/common/include/pcl/cuda/cutil_inline_drvapi.h

```diff
@@ -85,10 +85,15 @@ inline int _ConvertSMVer2CoresDrvApi(int major, int minor)
 // This function returns the best GPU based on performance
 inline int cutilDrvGetMaxGflopsDeviceId()
 {
-    CUdevice current_device = 0, max_perf_device = 0;
-    int device_count     = 0, sm_per_multiproc = 0;
-    int max_compute_perf = 0, best_SM_arch     = 0;
-    int major = 0, minor = 0, multiProcessorCount, clockRate;
+    CUdevice current_device = 0;
+    CUdevice max_perf_device = 0;
+    int device_count     = 0;
+    int max_compute_perf = 0;
+    int best_SM_arch     = 0;
+    int major = 0;
+    int minor = 0;
+    int multiProcessorCount;
+    int clockRate;
```

Comment:
Forgot to mention: You can reduce the scope of this variables, too.

Reply:
For some of them, sure.
Any reason why this is not in the codacy issues (the scope of the variable can be reduced)?

Reply:
Coday isn't perfect, no tool is.

Reply:
Done.